index.html

```diff
@@ -1879,6 +1786,99 @@ <h2 id="AudioContext">
             </p>
           </dd>
           <dt>
+            Promise&lt;void&gt; suspend()
```

Comment:
1. The current suspend() method definition moves from `BaseAudioContext` to `AudioContext`. (inserting)